app/bundles/CoreBundle/Form/RequestTrait.php

```diff
@@ -222,6 +222,11 @@ public function cleanFields(&$fieldData, $leadField)
             case 'email':
                 $fieldData[$leadField['alias']] = InputHelper::email($fieldData[$leadField['alias']]);
                 break;
+            case 'country':
```

Comment:
I don't see relation between PR and these lines, should I?

Reply:
This change was to accommodate a change ConnectWise made to their API. The API now returns a company reference, which is an array,  instead of just the name of the country as text.

Sorry, I forgot to mention that in the commit.

Regards,
Sagardeep Singh

Reply:
Just to be sure, this change does not break anything else. RequestTrait is used on many places.

Reply:
Sorry, I mean't a country reference. I have reverted the change to the request as I realized that the fields function has a ref type that would be using for this case.